### PR TITLE
[ci-utils] New package: ci utility scripts

### PR DESCRIPTION
**Pull request by y-lohse**
See the readme file for an explanation of what this PR does.

Note that I'm not 100% the `bin` trick will work out, but best way to find out is to publish the package!

### CODE REVIEW

**Comment by y-lohse** [comment]
[(𝐒) Build APK automatique - RAF 1/2](https://trello.com/c/gPoflW01/1693-%F0%9D%90%92-build-apk-automatique-raf-1-2)

File: CODEOWNERS [hunk 1]
```diff
@@ -6,6 +6,7 @@ packages/eslint-config-cozy-app @cpatchane
 packages/commitlint-config-cozy @kosssi @enguerran
 packages/browserslist-config-cozy @cpatchane @Crash--
 packages/cozy-app-publish @cpatchane @gregorylegarec @y-lohse
+packages/cozy-ci-utils @y-lohse
```
Comment (CPatchane) [inline CODEOWNERS]: Maybe add another one here to not be alone maintaining this package, if no one you can add me.

File: packages/cozy-ci-utils/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+## cozy-ci-utils
+
+A series of utility commands for our CI.
+
+### Post a Github comment
+
+`cozy-ci-utils-github "But says hi"` Will post the message `Bot says hi` to the current PR.
```
Comment (cedricmessiant) [inline packages/cozy-ci-utils/README.md]: typo: `cozy-ci-utils-github "But says hi` :arrow_right: `cozy-ci-utils-github "Bot says hi`

File: packages/cozy-ci-utils/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+Requires an env var named `GH_TOKEN` which contains a Github token with the necessary permissions. It is recommended to generate this token from the @cozy-bot account.
+
+### Install an configure Transifex
```
Comment (cedricmessiant) [inline packages/cozy-ci-utils/README.md]: typo: `and configure`

**Comment by ptbrowne** [comment]
I would drop the "utils" from the name of the package and from the binaries.

> Note that I'm not 100% the bin trick will work out, but best way to find out is to publish the package!

You can do `npm install my-package@file:///Users/y-lhose/my-super-duper-package` ;)

File: packages/cozy-ci-utils/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+## cozy-ci-utils
+
+A series of utility commands for our CI.
+
+### Post a Github comment
+
+`cozy-ci-utils-github "But says hi"` Will post the message `Bot says hi` to the current PR.
+
+Requires an env var named `GH_TOKEN` which contains a Github token with the necessary permissions. It is recommended to generate this token from the @cozy-bot account.
```
Comment (ptbrowne) [inline packages/cozy-ci-utils/README.md]: I would add a bit to precise that it only works on Travis.

File: CODEOWNERS (hunk 1 above)
Comment (y-lohse) [reply CODEOWNERS]: Bah, I don't mind :) Some other packages in the list have only one maintainer too!

File: packages/cozy-ci-utils/README.md (hunk 4 above)
Comment (y-lohse) [reply packages/cozy-ci-utils/README.md]: Meh, the first sentence of the readme says these scripts are for our CI, which is Travis — none of them will work outside of it.

File: CODEOWNERS (hunk 1 above)
Comment (CPatchane) [reply CODEOWNERS]: As you wish

File: packages/cozy-ci-utils/README.md (hunk 4 above)
Comment (ptbrowne) [reply packages/cozy-ci-utils/README.md]: We are using Jenkins also.

**Comment by ptbrowne** [comment]
LGTM

**Pull request merged**